### DISCUSSION TITLE
So long, September 👋  [Repositories Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#139750] [Repositories]
![Header for the Repositories category shows a dark purple abstract background with white text reading “Repositories Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a group of repositories. To the right, there is a GitHub logo in two-tone purple.](https://github.com/community/community/assets/144172738/06f2261b-e8f4-40a3-b92b-99cf29b86134)

As we finish out the month of September, we’ve got some great member contributions to highlight, multiple feedback opportunities, and some Universe excitement! 

## Wall of Fame 🤩 : These folks make our category so great !
* @dfosco: For giving an answer and some product feedback all in [one reply](https://github.com/orgs/community/discussions/136940#discussioncomment-10463559) on repository creation. Thanks @vedantwatpade for asking the question so clearly, the included screenshots were helpful, and for marking the answer ✅
* @shayan0v0n: [Their comment](https://github.com/orgs/community/discussions/137221) on why it’s helpful to make multiple branches in a very easy to understand form - even if you aren’t technical. Great work! 
* @jovanovic-djo : When a member asked why their branch wasn’t able to be deleted, @jovanovic-djo stepped up with a [heroic answer](https://github.com/orgs/community/discussions/138540#discussioncomment-10642814) that included examples and screenshots. Thanks for having the heart of a teacher ❣️ 

### 📣 Announcements: The latest updates in the Community 🆕
* **GitHub Docs**: We want to hear about your experiences using technical documentation, join @felicitymay in [her discussion](https://github.com/orgs/community/discussions/139073) to share and describe the last time you used technical documentation.  
* 🌟 **Put Your First Project in the Spotlight!**: Whether it's a simple "Hello World" script, a fun personal website, a creative game, or a unique solution to a problem, your project matters, and we'd love to showcase it! [Share your first GitHub project in our Discussion](https://github.com/orgs/community/discussions/137746). 

## 🗳️ Repository Releases: All updates, info, releases, etc that are Repo-related!
* **Push Rules are now generally available, and updates to custom properties**: You can now restrict pushes into your private and internal repositories and their forks, with push rules – a new type of ruleset. [Read through the details on our Changelog](https://github.blog/changelog/2024-09-10-push-rules-are-now-generally-available-and-updates-to-custom-properties/).
* **New commit details page (public beta)**: A new version of the commit details page is now available in public beta! This new page, which is enabled by default, lets you quickly understand and navigate the changes in a commit with improvements to filtering, commenting, and keyboard navigation. [Give feedback in our discussion](https://github.com/orgs/community/discussions/139005). 

### ⬆️ Level up: Resources to take your skills to new heights
* **How to build an open source metrics dashboard**:  Read [our blogpost](https://github.blog/open-source/maintainers/how-to-build-an-open-source-metrics-dashboard/) about how GitHub volunteers built an open source metrics dashboard for the World Health Organization and some best practices they picked up along the way.

### 🪐 Universe 2024: As the world’s fair of software, GitHub Universe 2024 will be an unparalleled gathering of the brightest minds, companies, and innovators in the industry. 
* **Get Tickets**: Visit the [Universe website](https://githubuniverse.com/) to purchase your tickets and plan your agenda!  
* **WATCH - Gear up for GitHub Universe 2024**: Want to know more before signing up to GitHub Universe 2024? [Watch our video](https://www.youtube.com/watch?v=Dmchc0dCOa4) to learn about the unique combination of technical discussions, practical solutions, and fun-filled activities we have in store for attendees. 

Will you be attending GitHub Universe? Let us know if you’ll be joining us in person or virtually!

**Comment by Jaziel-Ali** [comment]
Open pastor 
Juan Alberto Plata Gutierrez 
🪙

**Comment by SoyHary999** [comment]
[texto.txt](https://github.com/user-attachments/files/17294902/texto.txt)

**Comment by queenofcorgis** [comment]
[Our new discussion](https://github.com/orgs/community/discussions/142395) has the latest Repositories check-in, come give it a read!
